### PR TITLE
Update keys.txt with ref:FR:NAF, ref:FR:SIREN and ref:FR:SIRET

### PR DESCRIPTION
**Pull request by waterced**
Add support for ref:FR:NAF, ref:FR:SIREN and ref:FR:SIRET used in France.
Related to https://github.com/streetcomplete/StreetComplete/pull/5917

### CODE REVIEW

**Comment by waterced** [comment]
If I understand well, the script need to be run for this to be taken into account in SC.
I do not have the possibility to do so immediately

File: keys.txt [hunk 1]
```diff
@@ -648,8 +651,8 @@ district    // used in Liberia with healthcare, address-based
 sub_district    // used in Liberia with healthcare, address-based
 
 us.fo:Adressutal    // unspecified integer, see https://wiki.openstreetmap.org/wiki/Import/Catalogue/usfo , automatic translation from Danish says "Address number"; used for import syncing - see https://www.openstreetmap.org/changeset/1888617
-us.fo:Galdandi_frá  // discardable, see https://wiki.openstreetmap.org/wiki/Import/Catalogue/usfo , asked on https://www.openstreetmap.org/changeset/1888617 (means "valid from" date)
```
Comment (matkoniecz) [inline keys.txt:651]: weird that also that line got changed

Comment (waterced) [reply keys.txt:651]: Arg that's an encoding issue. I edited via GitHub, which converted to UTF-8 I think

**Comment by waterced** [comment]
New PR here 
https://github.com/mnalis/StreetComplete-taginfo-categorize/pull/27

**Pull request closed without merging**